sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -361,4 +350,81 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       assert(spark.sharedState.cacheManager.lookupCachedData(samePlan).isDefined)
     }
   }
+
+  test("SPARK-27248 refreshTable should recreate cache with same cache name and storage level") {
```

Comment:
As the test name suggested, please add storage level match, too. From Spark 3.0, we support to specify storage level..
```
scala> sql("CACHE TABLE testCacheSelect OPTIONS('storageLevel' 'DISK_ONLY') SELECT 1")
res0: org.apache.spark.sql.DataFrame = []
```

Reply:
Thanks. I will update it accordingly.

Reply:
BTW, may i know where we can find the Spark 3.0 SQL reference?  https://spark.apache.org/docs/latest/api/sql/index.html only lists the supported functions....

Reply:
IIRC, we stopped to publish snapshot docs. Only maven snapshot artifacts are published.